int32 gnssID
int32 svID
int32 cn0
int32 flags
